base/requirements.txt

```diff
@@ -6,6 +6,7 @@ colorama==0.4.3 # required for structlog
 connexion==2.6.0
 Flask-Opentracing==1.1.0
 Flask==1.1.1
+flower==0.9.3
```

Comment:
what is that for?

Reply:
they look pretty :rose:

Reply:
true

Reply:
`flower` is a celery monitoring tool - we have an example use in the docker-compose file (commented out).